src/main/java/org/folio/service/transformationfields/TransformationFieldsConfig.java

```diff
@@ -47,6 +49,8 @@
 
   //Holdings specific Fields
   INSTANCE_ID("instanceId", "$.holdings.instanceId"),
+  HOLDING_NOTE_TYPE("holdingNoteTypeId", "$.holdings.notes[?(@.holdingsNoteTypeId=={id}) && ?(!(@.staffOnly) || @.staffOnly == false))]", HOLDING_NOTE_TYPES),
```

Comment:
I think we should display the name of the note right, so it should be ".note" as well in the path